rclcpp/include/rclcpp/copy_all_parameter_values.hpp

```diff
@@ -77,6 +81,28 @@ copy_all_parameter_values(
   }
 }
 
+/// Load a list of parameters from a yaml parameter file.
+/**
+ * \param[in] yaml_name The name of the yaml file that needs to be loaded.
+ * \param[in] node_interfaces The list of variadic NodeInterfaces taken as a template to set parameters for
+ */
+template<typename NodeT>
+std::vector<rcl_interfaces::msg::SetParametersResult>
+load_parameters(
```

Comment:
this file name should be changed into something else like `parameter_interfaces.hpp`? `copy_all_parameter_values.hpp` sounds dedicated to `rclcpp::copy_all_parameter_values`?

Reply:
Yea I agree, I just didn't want to change it too fast. `parameter_interfaces.hpp` sounds good to me.